prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    const fullName = 'Alan Kim';
+    console.log('fullName:', fullName);
+    const numberOfPets = 1;
+    console.log('numberOfPets:', numberOfPets);
+    const likesVegetables = true;
+    console.log('likesVegetables:', likesVegetables);
+    const nothing = 'null';
```

Comment:
Here you'll want to leave out the quotes so the JavaScript engine doesn't read `'null'` as a string instead of `null`.